ssl/ssl_lib.c

```diff
@@ -1336,6 +1337,11 @@ int SSL_set_fd(SSL *s, int fd)
     }
     BIO_set_fd(bio, fd, BIO_NOCLOSE);
     SSL_set_bio(s, bio, bio);
+#ifndef OPENSSL_NO_KTLS
+    if (ktls_enable(fd)) {
+        printf("setsockopt failed %d\n", errno);
```

Comment:
push an error code, don't printf in the library.

Reply:
This is the same as in crypto/bio/bss_sock.c. I'll fix it throughout the patchset according to what we agree about there.